python/cudnn/gemm/cutedsl/grouped/unfused/_bf16_api.py

```
@@ -190,16 +237,20 @@ def _validate_offset_sequence(values: Tuple[int, ...], *, expert_cnt: int, tenso
+        if get_data_ptr(tensor) % 16 != 0:
             raise ValueError(f"{name} data pointer must be 16-byte aligned")
 
     @staticmethod
     def _validate_pointer_array_alignment(tensor: torch.Tensor) -> None:
-        if tensor.data_ptr() % 8 != 0:
+        if get_data_ptr(tensor) % 8 != 0:
             raise ValueError("b_ptrs data pointer must be 8-byte aligned")
 
-    @staticmethod
-    def _record_pointer_stream(b_ptrs: torch.Tensor, current_stream: cuda.CUstream) -> None:
+    def _record_pointer_stream(self, b_ptrs: torch.Tensor, current_stream: cuda.CUstream) -> None:
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)